entity-framework/core/providers/cosmos/index.md

```diff
@@ -92,7 +92,7 @@ Once configured the partition key property should always have a non-null value.
 
 ## Embedded entities
 
-For Cosmos owned entities are embedded in the same item as the owner. To change a property name use [ToJsonProperty](/dotnet/api/Microsoft.EntityFrameworkCore.CosmosEntityTypeBuilderExtensions.ToJsonProperty):
+Owned entities are embedded in the same item as the owner. To change a property name use [ToJsonProperty](/dotnet/api/Microsoft.EntityFrameworkCore.CosmosEntityTypeBuilderExtensions.ToJsonProperty):
```

Comment:
Can you please bring back "For Cosmos" here? A comma after that would be better though.

Reply:
Agreed and done.